lib/bolt/node/winrm.rb

```diff
@@ -73,9 +73,24 @@ def shell_init
   $startInfo.RedirectStandardOutput = $true
   $startInfo.RedirectStandardError = $true
 
+  $stdoutHandler = { $Event.SourceEventArgs.Data | Out-Host }
```

Comment:
Need to simulate in tests a scenario where this event-based async stream draining is necessary instead of the previous synchronous method of:

```powershell
$process.StandardOutput.ReadToEnd() | Out-Host
```

For instance, the following does *not* actually demonstrate a problem with streaming responses over winrm, as it already behaves as one would expect (note that the stdout responses come back over time separately as shown in `--debug` output):

```
2017-10-13T17:21:35.193116  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: Opened session
2017-10-13T17:21:35.193192  INFO y1qk6tjzdnxz9hr.delivery.puppetlabs.net: Running command: (1..10) | % { Start-Sleep -Seconds $_; $_ } | Out-Host
2017-10-13T17:21:35.193263  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: Executing command: (1..10) | % { Start-Sleep -Seconds $_; $_ } | Out-Host
2017-10-13T17:21:37.124894  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 1

2017-10-13T17:21:37.124945  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:21:39.072197  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 2

2017-10-13T17:21:39.072250  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:21:42.088645  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 3

2017-10-13T17:21:42.088693  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:21:46.086485  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 4

2017-10-13T17:21:46.086532  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:21:51.086842  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 5

2017-10-13T17:21:51.086889  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:21:57.088497  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 6

2017-10-13T17:21:57.088550  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:22:04.088515  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 7

2017-10-13T17:22:04.088569  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:22:12.145209  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 8

2017-10-13T17:22:12.145255  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:22:21.118397  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 9

2017-10-13T17:22:21.118465  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:22:30.597177  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stdout: 10

2017-10-13T17:22:30.597227  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: stderr:
2017-10-13T17:22:30.612767  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: Command returned successfully
2017-10-13T17:22:30.620994  DEBUG y1qk6tjzdnxz9hr.delivery.puppetlabs.net: Closed session
y1qk6tjzdnxz9hr.delivery.puppetlabs.net:

1
2
3
4
5
6
7
8
9
10


Ran on 1 node in 55.44 seconds
```